src/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I see multiple configurations for prettier. Refer .. `https://prettier.io/docs/en/configuration.html`

Reply:
fixed